lib/cfl/tests/arena.c

```
@@ -0,0 +1,354 @@
+static void bound_external_rounding_and_cache(void)
+{
+    struct cfl_arena *arena;
+    cfl_sds_t value;
+    char *payload;
+    size_t payload_size;
+    size_t used;
+
+    payload_size = 1024 * 1024;
+    payload = malloc(payload_size);
+    TEST_CHECK(payload != NULL);
+    memset(payload, 'x', payload_size);
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)